airflow/config_templates/config.yml

```diff
@@ -861,6 +861,13 @@
       type: string
       example: ~
       default: "-"
+    - name: access_logformat
+      description: |
+        Access log format for gunicorn webserver
```

Comment:
A link to documentation that describes possible format argument would be helpful.

Reply:
Added link to documentation